app/src/main/java/com/eis/geoCalendar/network/SMS/SMSStringEventNetworkManagerAdapter.java

```diff
@@ -0,0 +1,102 @@
+package com.eis.geoCalendar.network.SMS;
+
+import android.content.Context;
+
+import androidx.annotation.NonNull;
+
+import com.eis.communication.network.NetworkManager;
+import com.eis.communication.network.listeners.GetResourceListener;
+import com.eis.communication.network.listeners.InviteListener;
+import com.eis.communication.network.listeners.RemoveResourceListener;
+import com.eis.communication.network.listeners.SetResourceListener;
+import com.eis.geoCalendar.gps.GPSPosition;
+import com.eis.geoCalendar.network.SMS.ListenerAdapters.GetResourceListenerAdapter;
+import com.eis.geoCalendar.network.SMS.ListenerAdapters.RemoveResourceListenerAdapter;
+import com.eis.geoCalendar.network.SMS.ListenerAdapters.SetResourceListenerAdapter;
+import com.eis.smslibrary.SMSPeer;
+import com.eis.smsnetwork.SMSFailReason;
+import com.eis.smsnetwork.SMSNetworkManager;
+
+import java.util.ArrayList;
+
+/**
+ * Simple network manager Adapter for {@link SMSStringEvent}s that uses {@link com.eis.smsnetwork.SMSNetworkManager}
+ * as real network manager. Uses Singleton Design pattern since {@link com.eis.smsnetwork.SMSNetworkManager} is a singleton
+ * <p>
+ * This class was created because of conflicts with the defined interfaces and the available network dictionary's interfaces
+ *
+ * @author Turcato
+ */
+public class SMSStringEventNetworkManagerAdapter implements NetworkManager<GPSPosition, ArrayList<SMSStringEvent>, SMSPeer, SMSFailReason> {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A class which only has private constructors should be final](https://app.codacy.com/manual/CremaLuca/geoCalendar/pullRequest?prid=5077110)

Reply:
Really?